src/engine/client/cl_keys.cpp

```diff
@@ -599,7 +292,7 @@ void Console_Key( int key )
 	}
 
 	// ctrl-L clears screen
-	if (key == 'l' && keys[ K_CTRL ].down) {
+	if (key == Key::FromCharacter('l') && keys[ K_CTRL ].down) {
```

Comment:
I was thinking that it sucked that we have to call this function many times per frame for a constant value. Just looking at the definition again, it seems Key::FromCharacter is pretty simple. Can it be a constexpr?